src/replication.c

```diff
@@ -1234,10 +1251,7 @@ free_replica(replica_t *r)
 void
 close_fd(int epollfd, int fd)
 {
-	if (epoll_ctl(epollfd, EPOLL_CTL_DEL, fd, NULL) == -1) {
-		REPLICA_ERRLOG("epoll error for fd(%d) err(%d)\n", fd, errno);
-		return;
-	}
+	ASSERT0(epoll_ctl(epollfd, EPOLL_CTL_DEL, fd, NULL));
```

Comment:
Take care of it, macro will replace this code with _DO_NOTHING_ in production build.

Reply:
ok. Thanks.